macstadium-shared-2/main.tf

```diff
@@ -0,0 +1,153 @@
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_ip_address = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.jupiter_brain_prod_version}"
+  config_path = "${path.module}/config/jupiter-brain-prod-com-env"
+  env = "com-prod"
+  index = "${var.index}"
+  port_suffix = 3
+}
+
+module "jupiter_brain_staging_com" {
+  source = "../modules/jupiter_brain_bluegreen"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_ip_address = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.jupiter_brain_prod_version}"
+  config_path = "${path.module}/config/jupiter-brain-staging-com-env"
+  env = "com-staging"
+  index = "${var.index}"
+  port_suffix = 4
+}
+
+module "worker_com_staging_1" {
+  source = "../modules/macstadium_go_worker"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.travis_worker_staging_version}"
+  config_path = "${path.module}/config/travis-worker-com-staging-1"
+  vm_ssh_key_path = "${path.module}/config/travis-vm-ssh-key"
+  env = "com-staging-1"
+  index = "${var.index}"
+}
+
+module "worker_com_staging_2" {
+  source = "../modules/macstadium_go_worker"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.travis_worker_staging_version}"
+  config_path = "${path.module}/config/travis-worker-com-staging-2"
+  vm_ssh_key_path = "${path.module}/config/travis-vm-ssh-key"
+  env = "com-staging-2"
+  index = "${var.index}"
+}
+
+module "vsphere_janitor_prod_com" {
+  source = "../modules/vsphere_janitor"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.vsphere_janitor_version}"
+  config_path = "${path.module}/config/vsphere-janitor-prod-com"
+  env = "com-prod"
+  index = "${var.index}"
+}
+
+module "vsphere_janitor_staging_com" {
+  source = "../modules/vsphere_janitor"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.vsphere_janitor_version}"
+  config_path = "${path.module}/config/vsphere-janitor-staging-com"
+  env = "com-staging"
+  index = "${var.index}"
+}
+
+module "collectd-vsphere-common-2" {
+  source = "../modules/collectd_vsphere"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.collectd_vsphere_version}"
+  config_path = "${path.module}/config/collectd-vsphere-common-2"
+  librato_email = "${var.librato_email}"
+  librato_token = "${var.librato_token}"
+  env = "common-2"
```

Comment:
That looks correct to me. `pod-2` (and `pod-1` in the other one) might be better now that we've"standardized" on the "pod" wording, but with the current wording everywhere this looks right.